### PR TITLE
Anthony Tesija submission

### PR DESCRIPTION
**Pull request by atesija**
@tcollier 

Compile command: **g++ whats_my_name_again.cpp -o its**   
Run command: **./its**

### CODE REVIEW

**Review by tcollier — APPROVED** [review]
Nice!

```
% g++ whats_my_name_again.cpp -o its
% ./its 
ANTHONYTESIJA
```

Also, bonus points for the fancy `<details>` tag in your README!

**Comment by tcollier** [comment]
Also, this entry is eligible for the short but sweet category

```
% wc -c whats_my_name_again.cpp 
253
```

**Comment by atesija** [comment]
Looks like I don't have write access on the repo so I can't merge the pr. Can I get access or can someone merge for me?

**Pull request merged**
